### ISSUE TITLE
Download hotkey doesn't work after using Firefox menu via hotkey

### ISSUE THREAD

**Issue by Eupeptic1** [Simon-Tesla/RaccoonyWebEx#70]
**Summary**

If Raccoony has opened a submission page for you, _and_ you use Firefox's built-in Ctrl-D (bookmark page) hotkey _first_, _and_ you select a non-default folder for the new bookmark, _then_ Raccoony seems to be "deaf" to its hotkeys (such as D to download), _until_ you click on the page background.

If Raccoony has opened a submission page for you, and you use Raccoony's hotkey first, the Raccoony hotkey works as expected.  If you use the Firefox Ctrl-D hotkey first, but accept the default folder for the new bookmark, then the Raccoony hotkey also works as expected.

**Steps to reproduce**
1.  In Firefox, navigate to an image gallery on a Raccoony-supported site.
2.  Mouse over the raccoon-head icon, and click "Open all in tabs".
3.  Switch to one of the tabs that Raccoony just opened for you.
4.  Press Control-D to create a new bookmark.
5.  In the Firefox "new bookmark" menu, select a folder for a new bookmark, other than the default folder.
6.  In the Firefox "new bookmark" menu, click the "Done" button, to close the menu.
7.  Try the d Raccoony hotkey.

Expected result: Image downloads; Raccoony menu expands from the raccoon head and the blue "Download" button on that menu changes to the white "Downloaded" indicator.

Actual result: Raccoony doesn't do anything.

8.  (Workaround) Click anywhere on the background of the page.
9.  Try the d Raccoony hotkey again.

Actual result: Image downloads; Raccoony menu expands from the raccoon head and the blue "Download" button on that menu changes to the white "Downloaded" indicator.

Expected result:  Should not need to click on the page to get the Raccoony hotkey to work.

**Steps to _not_ reproduce**
1.  In Firefox, navigate to an image gallery on a Raccoony-supported site.
2.  Mouse over the raccoon-head icon, and click "Open all in tabs".
3.  Switch to one of the tabs that Raccoony just opened for you.
4.  Press Control-D to create a new bookmark.
5.  In the Firefox "new bookmark" menu, immediately click the "Done" button, to close the menu.
6.  Try the d Raccoony hotkey.

Actual and expected result: Image downloads; Raccoony menu expands from the raccoon head and the blue "Download" button on that menu changes to the white "Downloaded" indicator.

**Environment**
Debian 9.13 on amd64/x86_64, desktop
Firefox 78.3.0esr, 64-bit
Raccoony 1.2.3

I tried this on FA, Weasyl, and Shorpy, and it has the same behavior on all 3.  I _think_ it will happen for any Raccoony-supported site.

**Comment by Simon-Tesla** [comment]
I'll take a look, but since I hook up the keyboard shortcut handling as a keyboard event handler in the injected script on the page I'm guessing the issue is that since the focus is no longer in the page but on the browser chrome where the page script can't handle it when you perform that sequence of actions. In theory I could implement global keyboard shortcuts, but there are a number of requirements for them and I'd have to do more work to ensure they don't conflict with a built-in shortcut.

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/commands

**Comment by Eupeptic1** [comment]
Yeah, I'm not sure how common this case is in the Real World (tm).  I normally use the mouse to bookmark the page in Firefox, open the Raccoony menu and click the "download" button, and finally click the site's "favorite" button.  When I do that, I don't see this bug.  Occasionally, when I have a large number of submission pages to review, I use the hotkeys/shortcuts, because it's slightly faster; I did that earlier and that's when I saw it.
